reactor-netty-http/src/test/java/reactor/netty/http/client/WebsocketClientOperationsTest.java

```diff
@@ -88,4 +94,34 @@ private void failOnClientServerError(
 		                 .uri("/login")
 		                 .responseSingle((res, buf) -> Mono.just(res.status().code() + ""));
 	}
+
+	@Test
+	public void testConfigureWebSocketVersion(){
+		DisposableServer httpServer = HttpServer.create()
+				.port(0)
+				.handle((in, out) -> out.sendWebsocket((i, o) -> o.sendString(Mono.just("test"))))
+				.bindNow();
```

Comment:
I propose to check on the server that the version that was sent by the client is really `8`.
You can do that with checking this header `sec-websocket-version: 8`

Reply:
great, thanks for the tip!

Reply:
good idea! please do capture the header in `handle` above in order to verify it